packages/core/src/types.ts

```diff
@@ -44,6 +44,7 @@ export type NormalizedOutputOptions = {
   workspace?: string;
   target?: string;
   schemas?: string;
+  namingConvention?: 'camelCase' | 'PascalCase' | 'snake_case' | 'kebab-case';
```

Comment:
This is a normalized determined type definition.
For user input before it is normalized, please add the definition here.

https://github.com/orval-labs/orval/blob/master/packages/core/src/types.ts#L197

In a normalized and determined type definition, there is always a value, so `?` is not necessary.

```suggestion
  namingConvention: 'camelCase' | 'PascalCase' | 'snake_case' | 'kebab-case';
```